### ISSUE TITLE
Internet Gateway attached_to? test is using wrong status code

### ISSUE THREAD

**Issue by patrickli** [k1LoW/awspec#252]
In the `attached_to?` method for `InternetGateway`, test for `state == 'attached'` is wrong. The state should be `available`.

Funny tough that the official AWS ruby SDK is inconsistent in the [documentation](http://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/Client.html#describe_internet_gateways-instance_method).

In one place it says
```
resp.internet_gateways[0].attachments[0].state #=> String, one of "attaching", "attached", "detaching", "detached"
```

In another place it says
```
attachment.state - The current state of the attachment between the gateway and the VPC (available). Present only if a VPC is attached.
```

I've tested it and it only returns `available`.

**Comment by inokappa** [comment]
@patrickli

Thank you for your report.

>Funny tough that the official AWS ruby SDK is inconsistent in the documentation.

I already confirmed that content. :bowing_man:
I'll fix it.

Regards.

Yohei

**Comment by k1LoW** [comment]
Fixed by @inokappa ( v0.73.1 ).
